### PR TITLE
First steps

### PR DESCRIPTION
**Pull request by psychodromus**
fixes according to comments please check and give some best practices regarding *.js file structure
all your comments has been responded feel free to accept or keep commenting so we can create the best code ever :P

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -1,3 +1,8 @@
-console.log();
-"wiktor ssie";
-console.log();
+var cookieTotal=0;
```
Comment (psychowico) [inline main.js]: always use spaces around operators and assignments symbol - it's simply more readable, think about structure like that (proper js):
```js
var cookieTotal=0,b=3,x=321;
//vs
var cookieTotal = 0;
var b           = 0;
var x           = 0;

File: main.js [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+function incrementTotalCookies()
+{
+  cookieTotal++;
+  document.getElementById('cookieCounter').innerHTML = 'kliknales ' + cookieTotal + ' razy';
```
Comment (psychowico) [inline main.js]: u can make this code faster (just for learning purposes).
if you store `document.getElementById('cookieCounter')` in variable, at the beginning of running this script u can use it later in the moment. now, your code searching in DOM every time. using DOM is slowest operations in javascript.
(of course in this case it's very fast too, simple id looking)

File: main.js (hunk 1 above)
Comment (psychodromus) [inline main.js]: Done

File: main.js (hunk 2 above)
Comment (psychodromus) [inline main.js]: Transfered label outside function to speedup process

**Comment by psychowico** [comment]
looks nice! just look at PR stats - 18 added, 18 removed. it's means good refactoring :D

**Pull request merged**